Controllers/api.controller.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { fetchApi } = require("../Models/api.model");
+
+const app = express();
+
+exports.getApi = (request, response, next) => {
+  
+    response.status(200).send(fetchApi());
```

Comment:
.send() should send the final response to the client. Sending a function here means theres no guarantee what will be sent back, and is very unreliable if the fetchApi function changes at all in the future.

Reply:
I have  now required in the JSON file straight to the controller and now sending that as the response